src/javascript/JContent/ContentRoute/ContentLayout/ContentLayout.jsx

```diff
@@ -92,21 +90,6 @@ export class ContentLayout extends React.Component {
         return (
             <>
                 <div className={classes.root}>
-                    <ResizableDrawer
```

Comment:
Looks like you could remove this component (the ResizableDrawer) from the project.